examples/README.md

```diff
@@ -0,0 +1,4 @@
+# Pyro Examples
+All files in this directory must be run with the [`dev`](https://github.com/uber/pyro#installing-pyro-dev-branch) version of Pyro.  Note that you will need the `extras` dependencies to run the examples.
```

Comment:
This is pretty incorrect, and I'm not sure it's helpful. What is required is that the example version matches the corresponding pyro version. For example, the examples in master will need to run with pyro at master, and the examples in dev will need to run with pyro at dev.

Can you reword to something like "All files in this directory must be run with a matching version of Pyro"?

Reply:
sure though we don't maintain a `master` branch. it's more of "each example at a certain commit in the tree is guaranteed to work with pyro at that same commit".  they'd have to roll back to the latest release tag to use the latest release version. i'll reword it.